docs/source/tracking/tutorials/remote-server.rst

```diff
@@ -0,0 +1,260 @@
+
+To specify the backend store and artifact store, you can use the ``--backend-store-uri`` and ``--artifacts-store-uri`` options respectively.
+
+.. code-block:: bash
+
+    mlflow server \
+      --backend-store-uri postgresql://user:password@localhost:5432/mlflowdb \
+      --artifacts-destination s3://bucket \
+      --host 0.0.0.0 \
+      --port 5000
+
+Replace ``localhost`` with the remote host name or IP address for your database server in actual environment.
+
+Step 4: Logging to the Tracking Server
+--------------------------------------
+
+Once the tracking server is running, you can log runs to it by setting the MLflow Tracking URI to the tracking server's URI. Alternatively, you can use the :py:func:`mlflow.set_tracking_uri` API to set the tracking URI.
+
+.. code-block:: bash
+
+    export MLFLOW_TRACKING_URI=http://127.0.0.1:5000  # Replace with remote host name or IP address in an actual environment
+
+Thenm run your code with MLflow tracking APIs as usual. The following code runs training for an scikit-learn RandomForest model on the diabetes dataset:
+
+.. code-section::
+
+    .. code-block:: python
+
+        import mlflow
+
+        from sklearn.model_selection import train_test_split
+        from sklearn.datasets import load_diabetes
+        from sklearn.ensemble import RandomForestRegressor
+
+        mlflow.autolog()
+
+        db = load_diabetes()
+        X_train, X_test, y_train, y_test = train_test_split(db.data, db.target)
+
+        # Create and train models.
+        rf = RandomForestRegressor(n_estimators=100, max_depth=6, max_features=3)
+        rf.fit(X_train, y_train)
+
+        # Use the model to make predictions on the test dataset.
+        predictions = rf.predict(X_test)
+
+Step 5: View logged Run in Tracking UI
+--------------------------------------
+
+Our pseudo-remote MLflow Tracking Server also hosts the Tracking UI on the same endpoint. In an actual deployment environment with a remote tracking server, this is also the case. 
+You can access the UI by navigating to `http://127.0.0.1:5000 <http://127.0.0.1:5000>`_ (replace with remote host name or IP address in actual environment) in your browser.
+
+Step 6: Download artifacts
+--------------------------
+
+MLflow Tracking Server also serves as a proxy host for artifact access. Artifact access is enabled through the proxy URI ``mlflow-artifacts:/``,
+giving users access to this location without having to manage credentials or permissions of direct access.
+
+.. code-section::
+
+      .. code-block:: python
+
+          import mlflow
+
+          run_id = "YOUR_RUN_ID"  # You can find run ID in the Tracking UI
+          artifact_path = "model"
+
+          # Download artifact via the tracking server
+          mlflow_artifact_uri = f"mlflow-artifacts:/0/{run_id}/{artifact_path}"
+          local_path = mlflow.artifacts.download_artifacts(mlflow_artifact_uri)
+
+          # Load the model
+          model = mlflow.sklearn.load_model(local_path)
```

Comment:
Can we load a model using `runs:/{run_id}`?

Reply:
oh didn't know that URI is available, not listed in the original doc haha